src/Data/NormalKey.purs

```diff
@@ -101,6 +97,8 @@ instance ordNormalKey :: Ord NormalKey where
   compare (NormalKey "Shift") _ = LT
   compare _ (NormalKey "Shift") = GT
   compare (NormalKey "Enter") (NormalKey "Enter") = EQ
+  compare _ (NormalKey "Enter") = GT
+  compare (NormalKey "Enter") _ = LT
   compare (NormalKey x) (NormalKey y) = compare x y
```

Comment:
Ah; if you intend for `Enter` to come after every key, then you can't use exactly the technique I proposed, but probably some variant is possible; like, where you write an injection `NormalKey -> Maybe Int`, taking `Enter` to `Nothing`, and then say that `Nothing` is greater than `Just i`, and that `Just i` is great than `Just j` when `i > j`, etc.